### DISCUSSION TITLE
How to specify multiple Google OAuth client IDs?

### DISCUSSION THREAD

**Discussion by markszente** [supabase/supabase#12479] [Questions] [answered]
According to Google Cloud Console:

> A client ID is used to identify a single app to Google's OAuth servers. If your app runs on multiple platforms, each will need its own client ID.

Now, I'm planning to use Supabase for a cross-platform mobile app. Obviously, I need at least two different client IDs for the iOS and Android versions, not to mention the web. How can I do that?

**Comment by snendev** [comment]
I ran into this as well. Is there a roadmap for supporting multiple client IDs per provider? (I know very little about the complexity of this problem.)

**Reply by markszente** [reply, marked as answer]
They're [working on it](https://github.com/orgs/supabase/discussions/13218#discussioncomment-5435475).
